pkg/util/utils_linux.go

```diff
@@ -19,7 +20,8 @@ import (
 )
 
 var (
-	errNotADevice = errors.New("not a device node")
+	errNotADevice              = errors.New("not a device node")
+	regexVirtualTerminalDevice = regexp.MustCompile(`^/dev/tty[\d]+$`)
```

Comment:
Can you put this into a function, with a check for 

strings.HasPrefix(path, "/dev/tty")
And only compile if necessary rather then have every user of Podman pay for the compilation.

You can wrap it with a sync.Once() to only do it once.

We want to avoid compiling in init, because of speed of starting a container. (For automotive use cases.)

Reply:
Using `path.Match` or implementing the check only with strings.TrimPrefix() + strconv.Atoi() then becomes a great alternative over using a regex

Reply:
SGTM

Reply:
Sounds good! Will implement that tomorrow :)